servicex/models.py

```diff
@@ -94,27 +96,20 @@ def to_json(x):
                 'id': x.id,
                 'admin': x.admin
             }
-
         return {'users': list(map(lambda x: to_json(x),
                                   UserModel.query.filter_by(pending=True)))}
 
     @classmethod
     def delete_all(cls):
-        try:
-            num_rows_deleted = db.session.query(cls).delete()
-            db.session.commit()
-            return {'message': '{} row(s) deleted'.format(num_rows_deleted)}
-        except Exception:
```

Comment:
Did you delete the exception handling because the messages are useless? 
Again, changes like this would be easier to understand if they were in their own commit with a rationale

Reply:
In this case, the route simply returns the return value from the `UserModel.delete_all` method. If some exception were thrown during the try block, this is the resulting response:
`<Response 36 bytes [200 OK]> b'{"message": "Something went wrong"}\n'
Two issues with this:
- The status code should be 500, not 200
- The message is useless
Flask-RESTful/RESTX can be configured to handle any uncaught error with a generic 500 internal server error, which is exactly what we want in this case anyway, but saves us from repeating this logic in other places. I was under the impression that this was set up by default, but it looks like it's not, so I'll remove these changes and make a separate PR for app-wide error handling.